roles/cinder-common/templates/etc/cinder/cinder.conf

```diff
@@ -66,11 +66,22 @@ volume_clear_size = {{ cinder.volume_clear_size }}
 {% endif -%}
 {% if backend.san_ip is defined -%}
 san_ip = {{ backend.san_ip }}
+san_secondary_ip = {{ backend.san_secondary_ip|default("") }}
 san_login = {{ backend.san_login }}
 san_password = {{ backend.san_password }}
 {% if backend.nimble_subnet_label is defined %}
 nimble_subnet_label = {{ backend.nimble_subnet_label }}
+{% endif -%}
+{% if backend.name == 'vnx' -%}
+storage_vnx_pool_name = {{ backend.pool_name }}
+naviseccli_path = {{ backend.naviseccli_path }}
+storage_vnx_authentication_type = {{ backend.authentication_type }}
+initiator_auto_registration = True
```

Comment:
Sourcing multiple configurations files from /etc/cinder sounds like a good way to go.